app/src/components/media/Video.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { css } from "@emotion/react";
+
+interface VideoProps {
```

Comment:
Maybe we also add preload prop with a default value of "none" so that videos are only fetched when played